docker-compose.yml

```diff
@@ -390,6 +390,7 @@ services:
       dataProviderUrl: "http://poa-ganache:8545"
       dataRegistryAddress: "${POA_REGISTRY_ADDRESS}"
     volumes:
+      - ./creator-node/.clinic:/usr/src/app/.clinic
```

Comment:
This will be shared between nodes, I don't think that is intended behavior

Reply:
Oh ya not at all - I'm still trying to get a POC working. Ideally load testing would be a completely different container and not a replica